### DISCUSSION TITLE
Generally 2x XP Rate till 58

### DISCUSSION THREAD

**Discussion by SynaX1** [chromiecraft/chromiecraft#4826] [Suggestions]
I know it's been requested multiple times, 
but after playing on the server over the weekend, 
I've noticed that 2x rate doesn't break the gaming experience.

Leveling in WoW has never appealed to me (shame on me).

For my example, I can't use some things because my account is much older than 7 days and
in most cases you only find people when you play on the server and not before, 
that's why i think Recruit a Friend a very outdated version.

I'm sure if you raise the mob and quest XP to 2x (variable via command) till Level 58 it would appeal to more people and 
it wouldn't take away the appeal that ChromieCraft gives people.

**Comment by Cooties-cc** [comment]
It's just going to get denied again and again. There are plenty of options to decrease leveling speeds, even if you were "too late" to get RAF code for 2x exp till 58, every weekend is now 2x, and BGs got a boost in exp given. During the 59 to 60 bracket, I've got every class to 60 with ingame play time around 1-3 days depending on class, and rarely was it during weekends. Without utilizing 2x speeds, at a slow to moderate pace leveling, you could generally have a 60 by the end of the week. The harshest reality people need to understand about MMOs and this server in particular are 
1. MMOs cater to people that waste more time in them (by design they are a time sink)
2. CC server is designed to fix bugs, and not push content faster than it's fixed
3. If you do not agree with these 2 facts, then it's generally you that should make changes, and not to the game/server

There have been many accommodations CC server has made for QoL improvements, of such have been to ease leveling aspects. However as I have pointed out many times before about the requests to increase the exp rates, nothing done to make it faster will ever please these types of people, and they will only ever ask for more.

**Comment by SynaX1** [comment]
Your arguments don't make much sense.
Most people are tired of doing the same boring quests over and over again.

You don't change the content just because you make it easier for people to start playing here and 
so increasing the population is something good in my opinion.

Class Balancing > "Level Content" fix

**Comment by Cooties-cc** [comment]
`Most people are tired of doing the same boring quests over and over again.`
This is a heavily overused statement about the process to level a character. Not only are there methods in place to reduce the time to level, but there are options like RDF and BGs that can completely bypass doing a single quest. Repetitive tasks are what a game is, more so an MMO. 

`Faster leveling means more population`
Woefully wrong. High population does not directly correlate with faster leveling or instant caps. Looking at player interest and retention, those who prefer/seek faster rates or instant cap servers, are most likely to quit the server first. Having shorter investment time into any project will only lead to having lower retention rates. You will also attract a higher volume of lower value social players, in that they do not care about their accounts since it didn't require any effort to invest with, and are objectively abhorrent to play with or talk to. 

`Class Balancing > "Level Content" fix`
https://www.chromiecraft.com/en/about/
I think you may want to read (if you even did) what the mission statement is about this server. You may have your own preconceived notions about what _you_ want from a server, but that doesn't mean every server will cater to it. 

If faster leveling was such a desired attribute to securing the success of a servers population and health, why did ZERO classic retail wow servers make a single high rate leveling server? Even after moving onto the next expansion, why did the previous expansion content not receive a leveling boost?

What draws in a large crowd, and retains that crowd, is a good environment. Comprising of a good online and offline social space, a server that has high uptime with no crashes or lag spikes, a good working server in that has little to no bugs, a GM/dev team that is transparent about fixing things while communicating with the public and not doing shading backdoor deals, and overall something people want to play.

**Reply by ganjjo** [reply]
And this is why your server will continue to be a dead server that only has 500 max online.

>Not only are there methods in place to reduce the time to level, but there are options like RDF

Good luck with RDF when you sit in que for hours and it never pops. 500 (max) players doesnt help my level 20 character complete dungeons. Please tell us again about these other options. There are ZERO options to reduce the leveling time. It takes X amount of XP to level and this server has 0 ways to decrease that amount, except for weekends where you ONLY add 2x XP for quest completion. BGs. If I wanted to PvP I wouldnt be hiding on a PvE server with the rest of you. XP boost for PvP is fine but an XP boost for PvE will break the universe....Can you explain why you gave PvPers a XP boost but for the PvErs playing on a PvE server its not acceptable?

>There have been many accommodations CC server has made for QoL improvements, of such have been to ease leveling aspects. However as I have pointed out many times before about the requests to increase the exp rates, nothing done to make it faster will ever please these types of people, and they will only ever ask for more.

You increased PvP XP because you want it but when PvErs ask you say no because you dont want it.

> why did ZERO classic retail wow servers make a single high rate leveling server

This is by far the most pathetic attempt I have ever seen to refute some ones point. Blizzard created Classic, not classic + or classic but faster. Chromiecraft IS NOT a retail server, stop comparing it to one.

> why did the previous expansion content not receive a leveling boost

It did but it doesnt support your lies so you conveniently left it out,
**One major new feature is players under level 60 receiving the “Adventure Awaits” buff which will increase the experience gained from turning in quests by 40%. The buff will also grant additional bonus experience from Group and Dungeon Quest turn-ins, as well.** Additionally, the Looking for Group tool currently in Burning Crusade Classic will become available in SoM.

> What draws in a large crowd, and retains that crowd, is a good environment.  Comprising of a good online and offline social space, a server that has high uptime with no crashes or lag spikes, a good working server in that has little to no bugs

A good environment STARTS with lots of people, 500 is NOT a lot of people.  You need lots of people so everyone can run dungeons when they level but its not possible here cause most people are max level. Higher rates gets people play alts which means new players can run dungeons instead of sitting in a que for 6 hours and not completing anything. If rates were higher to 60 then we could get up to level faster and play the CURRENT expansion.

This server is far from bug free. And its bugs that have been fixed in other cores for almost a DECADE. Some basic bugs that shouldn't exist. 9/10 times I click on an NPC no text shows. I have to click on NPC, deselect NPC, click NPC again and it finally shows. How do you get into TBC and this still exists? Bags. You cannot swap bags if the one youre swapping has items. Blizzard fixed this by TBC and your on WOTLK core and it is present. I cant remember that last P server I played where this was present. 

You only seem interested in shooting down every suggestion that doesnt support your vision, whatever that is.

**Reply by Cooties-cc** [reply]
As I said in my posts above, RaF codes give 2x exp till 58, and any new player can obtain these codes by asking in world or discord very easily. They last for 30 days or when a character hits level 58. This is primarily PvE as it gives rested exp, which affects kills, quests, and rdf completions. Should affect BGs exp too, but can't recall. 
https://www.chromiecraft.com/en/recruit-a-friend-is-available-now/

RDF won't be solved with higher populations on the server, since majority of population on a server is at or near max cap. 
![image](https://user-images.githubusercontent.com/62584589/214379235-6c7c6991-1762-4acd-92d5-bf4c26650d1b.png)
I've been running the census for quite some time, and pruned my data removing 30+ day inactives to get this. Around 50% of the total population sits between 60 and 64. Removing the level 1s, the avg amount of players sitting in the leveling stages sits around 15-40ish, so say 27 avg for each lvl 2 - 59. If max population brought in players equally to each bracket, and assuming the server can match retail caps around 5k co-current players, that's a 10x increase in overall populations. However you then cut out 50% of that to "endgame" players, so 2500 remain to spread across 2-59, which is around 2500/58; 89.2. So you bring up the avg player per lvl bracket from 27 to 89. This is also assuming you can maintain 5k co-current players, which servers have peaks and valleys of logins. It would hardly make a dent in your RDF ques while leveling, even negating some cap players leveling alts instead of not playing or afking in a city. 

The question pertaining to the high rate server options blizzard could have done for the classic servers is a very relevant argument. The crux of this discussion, if you want to call it that, is that higher rates for a server will improve server population or growth. This means it would be more popular choice since higher population means more people want that, which means more people would want to play, which means more subs, thus more money for blizzard to take. Yet they kept it at 1x, and gave no options for alternative servers at higher rates (2x+). Higher rates would only help a company hellbent on getting more players, but didn't. Devils advocates says it was to prolong content so subs last longer. However content was already time gated, and majority of players were at/near the end of that content cycle waiting already doing nothing. It simply makes no sense on a server like this to rush people to the endgame content to sit and do nothing, which instead they could still be reporting problems as they level 1x (more content tested). They still have the increased rates options too.  

I also admit I didn't know about the SoM buff change (2021-11-09) that was applied a couple of months after TBC classic launched. I wasn't lying about it, just didn't see it show up on news cycles or from people talking. I didn't play SoM, but heard they had changes. The original buff design was only 40% to *quest* exp, not all exp. Months later changed to the 100% version via hotfix (2022-02-15). Although, with these changes to exp rates, did the server get a flood of new players or higher rates of lower content accessible? Curious. 

Your ending statements also make no sense. 500 avg active players online (currently 380) is actually quite a lot given the type of server this is (PTE RP-PvE bracketed bug testing). The server I hear most pointed to is warmane, which lies about server pops, and has been around since 2013 (it was called Molten WoW before rebrading to Warmane). It's understandable to say even with lies, they have good pop, but also have been around for 9 years almost. Not many other servers (going by pserver lists scrapers) have a bolstering population either. CC is still early in the actual client development, and has only existed for (assuming by when the github was created 2020-12-21) about 2 years. It also doesn't make sense that you advocate for higher rates so that more people will create alts at cap, but then say *get up to level faster and play the CURRENT expansion*. So do you want higher rates for more alts, or to skip "trivial" content to sit at endgame doing nothing? Very odd. Not to mention you came across problems and bugs, and *didn't report them*, yet continue to complain about them? 
![image](https://user-images.githubusercontent.com/62584589/214397593-88edbcae-6c65-42af-a0c6-4f0dfc5b8941.png)
You joined github just yesterday, and I assume was to only log on to try and flame me, as if you know anything about what I do here. Or at worst, are an alt account. Just doesn't make sense to me to try and paint my as a villain when I've only voiced my stances about high rates on and the effects they have on private servers.

**Comment by Seperest** [comment]
My suggestion is for them to add an item to the database. A shirt with no appearance, Bind on Account, and Doubles exp gained from all sources. Make it unique-equipped. When a character reaches lv70, their Chromie mail can include this shirt. That way, the player can choose to wear it or not. When level cap gets raised, remove the item from everyones inventory/characters, and remove it from lv70 mails and add it to current max mails.

**Comment by cachocarne** [comment]
Since I don't play on Chromie anymore, you may say my opinion doesn't count, but maybe some people find it interesting that me and my friends stopped playing the moment the game started to do custom adaptations to make things more convenient. Crossfaction pvp lead to crossfaction dungeons and raids to crossfaction guilds. Adding TBC items before they should be in the game, changing rates and allowing people to get items that shouldn't be available to players... The phylosophy of Chromiecraft and Azerothcore got kinda manipulated there and by trying to appease a few players, it neglected a whole bunch too. Honestly, I would love to step up again, but that would only happen if realm would be like before, with slow progression, attention and care for each quest and tiny script. And if to add any custom changes, it should be much more on the opposite direction, making the game more challenging (needless to say, HC mode is booming on live Classic servers) and interesting.
